class.application-passwords.php

```diff
@@ -340,10 +340,14 @@ public static function show_user_profile( $user ) {
 		wp_enqueue_style( 'application-passwords-css', plugin_dir_url( __FILE__ ) . 'application-passwords.css', array() );
 		wp_enqueue_script( 'application-passwords-js', plugin_dir_url( __FILE__ ) . 'application-passwords.js', array() );
 		wp_localize_script( 'application-passwords-js', 'appPass', array(
-			'root'       => esc_url_raw( rest_url() ),
-			'namespace'  => '2fa/v1',
-			'nonce'      => wp_create_nonce( 'wp_rest' ),
-			'user_id'    => $user->ID,
+			'root'       		=> esc_url_raw( rest_url() ),
+			'namespace'  		=> '2fa/v1',
+			'nonce'      		=> wp_create_nonce( 'wp_rest' ),
+			'user_id'    		=> $user->ID,
+			'translations'	=> array(
+				'revoke_password'				=> esc_attr__( 'Are you sure you want to revoke this password? This action cannot be undone.' ),
```

Comment:
Tabs to indent, spaces to align.  Your whitespace here before the `=>` is using tabs, not spaces and is waaaaay too big.